setup.cfg

```diff
@@ -67,6 +67,8 @@ exclude =
     *.egg
     build
     temp
+    # todo: some strange failure
+    ./tests/callbacks/test_pruning.py
```

Comment:
Any idea about this? I can have a look... but should have anything to do with torchtext 🤔

Reply:
no, it is unrelated and even the check details can't be found in codebase